spec/request/csrf_request_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+# This file is for exercising routes that should require CSRF protection.
+# It is very much a WIP
+
+# TODO:
+# Lighthouse API endpoints
+# Check routes.rb for other `POST` routes
+
+require 'rails_helper'
+
+RSpec.describe 'CSRF scenarios', type: :request do
+  before do 
+    get(v0_maintenance_windows_path)
```

Comment:
innocuous route chosen for setting the CSRF token in the cookie

Reply:
```suggestion
    # innocuous route chosen for setting the CSRF token in the cookie
    get(v0_maintenance_windows_path)
```
😄